Fix repeated commands object in Import Inline Hook reference (#3018)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/reference/import-hook/index.md

1. From the Settings column on the left side of the screen, select **To Okta**.

> 1. In the **Inline Hooks** section, click the **User Creation** drop-down menu. Any Inline Hooks you have registered will be listed. Select the one to use.

Reviewer: ```suggestion
1. In the **Inline Hooks** section, click the **User Creation** drop-down menu. Inline Hooks that you've registered appears in the list. Select the Inline Hook to use.
```
